csds-client/main.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"envoy-tools/csds-client/client"
+	"fmt"
+)
+
+func main() {
+	_, error := client.New()
```

Comment:
As per comment earlier, this does not make sense to create a New client and not use it. We should use the client and run the request from the main

Reply:
Solved. It will call `c.Run()` after `New()`.